app/code/Ludmila/LSNewClasses/view/frontend/layout/homework11_homework11_index.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<page layout='1column' xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd" >
+    <head>
+        <title>Homework11_OOP</title>
+    </head>
+    <body>
+        <referenceContainer name="content">
+            <container name="homework11_wrapper" htmlClass="homework11_wrapper" htmlTag="div">
+                <block class="Ludmila\LSNewClasses\Block\Homework11" template="Ludmila_LSNewClasses::homework11_block.phtml" name="homework11" />
```

Comment:
There is no class for block Ludmila\LSNewClasses\Block\Homework11, but there is Ludmila\LSNewClasses\Block\Homework11\HomeworkOOP. Did you wanted to use it? Please correct, doesn't work now